rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -672,6 +677,16 @@ rmw_node_t * rmw_create_node(const char * name, size_t domain_id)
   }
   memcpy(const_cast<char *>(node_handle->name), name, strlen(name) + 1);
 
+  node_handle->namespace_ =
+    static_cast<const char *>(malloc(sizeof(char) * strlen(namespace_) + 1));
```

Comment:
Yeah, I'll change it. However, `rmw_fastrtps` uses malloc in lots of places (since we didn't write it originally). Also the idea behind `rmw_allocate` is sort of a dead end in my mind. We should instead update `rmw` to use the allocator structure like `rcl` does.